fury/animation/interpolator.py

```diff
@@ -0,0 +1,348 @@
+import numpy as np
+from scipy import interpolate
+from scipy.spatial import transform
+from fury.colormap import rgb2hsv, hsv2rgb, rgb2lab, lab2rgb, xyz2rgb, rgb2xyz
+
+
+class Interpolator:
+    def __init__(self, keyframes):
+        super(Interpolator, self).__init__()
+        self.keyframes = keyframes
+        self.timestamps = []
+        self.min_timestamp = 0
+        self.final_timestamp = 0
+        self._unity_kf = True
+        self.setup()
+        self.id = -1
```

Comment:
What the IDs are used for?

Reply:
It was for shaders to send it as a uniform. should I remove it from the current version?